packages/dd-trace/test/config.spec.js

```diff
@@ -817,14 +836,14 @@ describe('Config', () => {
     const config = new Config({
       appsec: {
         enabled: true,
-        blockedTemplateHtml: path.join(__dirname, 'DOES_NOT_EXIST.html'),
-        blockedTemplateJson: path.join(__dirname, 'DOES_NOT_EXIST.json')
+        blockedTemplateHtml: 'DOES_NOT_EXIST.html',
+        blockedTemplateJson: 'DOES_NOT_EXIST.json'
       }
     })
-    expect(config.appsec.blockedTemplateHtml).to.be
-      .equal(path.join(__dirname, '..', 'src', 'appsec', 'templates', 'blocked.html'))
-    expect(config.appsec.blockedTemplateJson).to.be
-      .equal(path.join(__dirname, '..', 'src', 'appsec', 'templates', 'blocked.json'))
+
+    expect(log.error).to.be.called
```

Comment:
This is really not good coverage, please use more precise asserts like
```
expect(log.error).to.have.been.calledTwice
expect(log.error.firstCall).to.have.been.calledWithExactly()
expect(log.error.secondCall).to.have.been.calledWithExactly()
```
or something like that

Reply:
yes, but have we decided the error message?

Reply:
I think this is adequate: `dd-trace: Error reading file /not/found/file/path. Is the 'DD_APPSEC_RULES' environment variable correct?`